# Literature Review in Epigenetic and Gene Regulation

## 1. Overview
1. The regulation of gene expression is primarily encoded in /cis/and is directed by transcription factors [1].

## 2. Fundamental questions
1. How epigenetic modifications are interpreted and inherited and 2) how they influence genomic output before, during or after a cell­-fate transition are fundamental questions to our understanding of development. 

2. How DNA methylation is globally and locally modulated, and for what purpose, remain compelling questions, as only a few robust and general rules have been formulated. 

## 3. DNA methylation
1. Methylation of the fifth position of cytosine is one of the best studied and most mechanistically understood epigenetic modifications and is well conserved among most plant, animal and fungal models[2]. 

2. In mammals, cytosine methylation is primarily restricted to the sym­metrical CpG context [3,4].

3. Three conserved enzymes, DNA methyltransferase 1 (DNMT1), DNMT3A and DNMT3B, are responsible for its deposition and maintenance and are essential for normal development [5,6].

4. Mammalian genomes are globally CpG­ depleted and, of the roughly **28 million CpGs in the human genome**, **60–80% are generally methylated**. **Less than 10% of CpGs occur in CG­ dense** regions that are termed **CpG islands**; These are **prevalent** at transcription start sites(TSS) of **housekeeping** and **developmental** regulator genes, where they are largely resistant to DNA methylation[7]. 

5. DNA methylation has important roles in the maintenance of **Pericentromeric repeats, and Transposable elements** genomic components that may reflect its most conserved function across species. (Figure 3, 4)

### 3.1 Region Specific DNA Methylation

#### 1) Unmethylated non-promoter CpG Island

1. Nearly half of the unmethylated CpG islands identified in mam­malian genomes do not occur at annotated promoters, although many of these so­called ‘**orphan**’ CpG islands show similar epigenetic features [29]. 


#### 2) Promoter unmethylation

Most CpGs in mammalian genomes remain methylated during development, but CpG islands found at promoters of many housekeeping or developmentally regulated genes are constitutively **hypomethylated**. 


![Figure 1](https://github.com/Read-Lab-UCI/Epigenetics/blob/main/Figures/Fig1.png)

Figure 1:  **Most transcription start site (TSS)-associated CpG islands are protected from DNA methylation**. Components that confer this protection include: 
* 1). Transcription factor (‘TF’ in the figure) binding
* 2). Nucleosome exclusion
* 3). Histone H3 lysine 4 (H3K4) methyltransferases, such as SET domain containing 1A (SETD1A; recruitment of which is directed by CXXC finger protein 1 (CFP1)) or MLL proteins
* 4). Active transcription may also inhibit DNA methylation by forming DNA–nascent RNA helices, which induce R-loops of single-strand DNA (ssDNA) that exclude/de novo/methylation
* 5). The presence of catalytic enzymes associated with DNA demethylation, such as the TET enzymes or thymidine DNA glycosylase (TDG), may prevent aberrant methylation. (Figure from  Fig1a of Smith, Z. D., & Meissner, A at al 2013) [1]

1. **TF binding and even TF binding sequences are sufficient to maintain local hypometh­ylated state, both in CGI promoters or distal enhancers** which requires that DNMTs be actively and continuously excluded, even without measurable transcription [32]. Evidences are:

	* CpG islands can progressively accrue heritable methylation if they are truncated or depleted of known transcription factor binding sites [30, 31].
	* Transfer of an SP1 binding site into an endogenously methylated locus induced appreci­able local demethylation, confirming the dominance of transcription factor binding over DNA methylation in this context [30].
	* Low-methylated regions (LMRs) are occupied by cell-type-specific DNA-binding factors and their binding is necessary and sufficient to create LMRs in mouse neuronal and stem cells [26].

2. **Histone modifications and variants** are also significant in maintaining hypomethylated states at CpG islands.

	* **CFP1** (CXXC finger protein 1, also known as CXXC1) recruits histone H3 lysine 4 (**H3K4**) methyltransferases and is sufficient to main­tain ectopic, CG­rich transgenes that lack promoter features or transcription in an unmethylated state [33]. However, CFP1-­knockout cells lose local H3K4 tri­methylation (**H3K4me3**) without changes in expression or promoter DNA methylation [34].  This could reflect a difference between /de novo/ assembly of euchromatin and the retention of redundant protective mechanisms at already active loci. (See Figure 1)

	* Binding of the MLL family H3K4 methyltransferases protects promoters of developmental genes from DNA methylation, and this is also likely to be instructed through their CXXC domains [35]. (See Figure 1)
	
	* The **DNMT3** enzymes each contain an ADD domain (ATRX–DNMT3–DNMT3L)  that recognizes unmodified H3 and is allosterically inhibited by  **H3K4 methylation** [36, 37]. (See Figure 1)

	 * The histone vari­ant **H2A.Z** is strongly enriched at unmethylated, active promoters [38].  When DNA methylation is chemically inhibited by azacytidine (a hypomethylating agent), H2A.Z­ containing nucleosomes encroach into adjacent regions [39].  This supports a model in which epigenetic modifications and histone variants associated with transcription (such as H2A.Z) may be limited to their functional targets by surrounding DNA methylation [38, 39]. 

3. Many housekeep­ ing gene promoters have an asymmetric GC distribution downstream of their transcriptional start sites [43]. This GC skew may inhibit  DNMT3s recruitment by forming a single­ stranded DNA loop (known as an R­loop), through complementary base pairing between nascent RNA and the template strand [43]. Inversion of this GC asymmetry negates RNA–DNA helix formation and induces de novo methylation [43].  (See Figure 1)


4. CpG island promoters are not usually repressed by DNA methylation and are instead silenced by **H3K27 methylation**, which may also protect them from spurious DNA methylation [40, 41]. 

#### 3) De novo methylation at repressed promoters


![Figure 2](https://github.com/Read-Lab-UCI/Epigenetics/blob/main/Figures/Fig2.png)
 
Figure 2. Stable silencing of promoter regions requires recruitment of repressive transcription factors, which direct the recruitment 
of the chromatin remodeller LSH, linker histone H1, heterochromatin protein 1 (HP1), H3K9 methyltransferases (the G9A–GLP complex is shown) and /de novo/ DNA methyltransferases, frequently 
in that order. At germline gene promoters, DNMT3B is more directly involved in gene silencing than at other genes and acts downstream of transcriptional repressors (E2F6 is shown).  (Figure from  Fig1b of Smith, Z. D., & Meissner, A at al 2013) [1]

Repressed promoters, particularly those of lower CpG density, acquire DNA methylation during develop­ment.  The requirement of both H3K9 methylation and DNA methylation for complete, stable promoter silencing suggests a model in which H3K9 methylation initiates heterochromatin formation and DNA methylation ensures long­term silencing [51]. (See Figure 2). 

* Targeting: TFs such as RAR(retinoic­acid­response nuclear recep­tors) in the context of embryonic stem cell (ESC) differentiation often participate in proper targeting [44]. 

* Participants: 1) Both  **DNMT3A** and **DNMT3B** target promoters in complex with other **epigenetic repressors**, including 2) histone deacetylases (**HDACs**), 3) methyltransferases associ­ated with repressive **H3K9** methylation (H3K9 dimethyltrans­ferase **G9A**) and also with **nucleosome remodeller** (LSH, ) for heterochromatin assembly through interactions with epigenetic repressors.  

* The nucleosome remodeling may provide nucleosomes in previously depleted regions to act as a **template** for epigenetic silencing [45, 46, 47]. 

- LSH is co­recruited with the H3K9 dimethyltrans­ferase G9A (also known as EHMT2) in complex with DNMT3A or DNMT3B [48, 49]. The catalytic activity of G9A stabilizes and accelerates repression, but its bind­ ing activity is often sufficient for DNA methylation [49, 50]. 

#### 4)  Pericentromeric repeats silencing

Most of the sequence in mammalian genomes is non­coding but includes many features with latent transcriptional potential, includ­ing **pericentromeric repeats** (which **instruct centro­meric assembly**) and **parasitic repetitive elements**. 

Pericentromeric minor and major satellite ele­ments extend from the centromere in thousands to tens of thousands of tandem copies [52]. These elements have latent transcriptional potential, the repression of which is essential for proper chromosome alignment, segregation and integrity during mitosis. 

* **SUV39H1** deposits H3K9 methylation, directs **DNMT3B** to major satellites and is sufficient for silencing at these regions [52, 53]. 

-  **DNMT3B** is directly recruited by CENPC (Centromere protein C; also known as CENPC1) and functions down­ stream of pericentromeric nucleosome assembly, which is instructed by CENPB [55].  **DNMT3B** can be retained at these regions through **metaphase**, which suggests a con­tinued requirement for silencing throughout **mitosis** [54].  These data highlight a crucial, specific role for **DNMT3B recruitment** in maintaining **centromeric** proximal **heterochromatin** to facilitate proper **cell division**. 


![Figure 3](https://github.com/Read-Lab-UCI/Epigenetics/blob/main/Figures/Fig3.png)
 
Figure 3. Pericentromeric repeats are predominantly targeted by DNMT3B. At major satellites, DNMT3B is secondary to the activity of H3K9 methyltransferases SUV39H1 and SUV39H2 (shown as SUV39H). DNMT3B may be more immediately involved in minor satellite silencing by directly interacting with the centromeric nucleosome. 


Knockout Experiments:
-  DNMT3B- ­null cells exhibit hyperacetyla­tion at minor satellite repeats, leading to chromosomal mispairing and lagging anaphase bridges during mitosis [54]. 

#### 5) DNA methylation in Transposable elements

Endogenous transposable ele­ ments constitute nearly 40% of mammalian genomes and consist of three major classes: long interspersed nuclear elements (**LINEs**), short interspersed nuclear elements (**SINEs**) and long terminal repeat (**LTR**)­ containing endogenous retroviruses [56]. 

Full­ length LINE and LTR elements encode strong promoters that must be constitutively repressed to prevent their activity, and generally these regions are constitutively hypermethylated. However, in ESCs, DNMT1 activity alone is often not sufficient to maintain DNA methyla­tion stably at these sites, and DNMT3 enzyme recruit­ment and activity are also required [57].  

* Process and mechanism: 

	1. **LTR** sequences are silenced by **TRIM28**(tripartite-­motif-­containing protein 28) - ­mediated recruitment of the **H3K9 meth­yltransferase SETDB1**.  

	2. The histone methyltransferase activity of SETDB1 acts upstream of DNMT recruit­ment, and DNA methylation appears to function as a secondary stabilizer [58]. 

	3.  Mechanistically, silencing also involves G9A to initiate heterochromatin assembly and DNMT3­like (DNMT3L) to recruit de novo methyl­ transferases [50, 59, 60].


![Figure 4](https://github.com/Read-Lab-UCI/Epigenetics/blob/main/Figures/Fig4.png)
 
Figure 4. Repetitive elements such as long terminal repeat (LTR)-containing retroelements and long interspersed nuclear elements (LINEs) are also silenced by DNA methylation, although for LTRs, DNA methylation acts downstream of zinc finger protein (ZFP)-based recruitment of TRIM28 (also known as TIF1β) and the H3K9 methyltransferase SETDB1. **The mechanisms for LINE silencing are less understood**, but these elements exhibit enriched hemimethylation and hydroxymethylation, which may be countered by de novo methyltransferase recruitment and activity.

* Dynamics in development

The components required for de novo repetitive element silencing are largely restricted in their expression to discrete developmental windows, including early embryogenesis (and thus also ESCs). 

In adult tissues, the /de novo/silencing machinery is often not present, and DNA methylation is crucial to direct continued repression by recruiting **MECP2**(methyl­-CpG­-binding protein 2) in complex with histone deacetylases (**HDACs**) [61-65]. 

Developmental windows in which repeat activity is prevalent are restricted to the early embryo and during germline development, particularly during gametogenesis, in which a unique mode of epigenetic regulation involves PIWI­interacting RNAs (Details See BOX2 in Ref [1]). 

#### 6) Intragenic region methylation

1. Those CpG islands that occur at intragenic regions are more frequently methylated during development and may contribute more nuanced regulatory functions [29].

#### 7) Targeting imprints

DNA methylation is the classically assigned instructional modification for germline **ICRs** (imprint control regions), although mounting evidence implicates other epigenetic modifiers in guid­ing DNA methylation to imprinted loci. 



### 3.2 DNA methylation maintenance
 

#### 1) DNMT1
1. The DNA methyltransferase 1 (/Dnmt1/) transcript is constitutively expressed in dividing cells and is most abundant on entry into S phase owing to cell‐cycle‐specific transcription factors[8]. 

2. **DNMT1 is recruited** to DNA replication sites **through direct interactions** with proliferating cell nuclear antigen (**PCNA**) and **UHRF1** (also known as NP95), **which binds hemimethylated DNA** via its SET‐ and RING‐associated (SRA) domain [9, 10, 11].  

3. Core PCNA–UHRF1–DNMT1 interactions operate within a larger complex that includes chromatin‐associated enzymes that also regulate DNMT1 through opposing post‐translational modifications. The histone acetyltransferase TIP60 and methyltransferase SET7 target  DNMT1 and trigger polyubiquitylation, which regulates recruitment and degradation [15, 16]. **These modifications are opposed during the peak of DNMT1 activity by HDAC1**, in complex with DNA‐bound DNMT1 and presumably through histone **demethylases** such as **LSD1**  [17, 18].


#### 2) UHRF1

1. **UHRF1 directs DNMT1 to hemimethylated sites**, binding specifically to the parental, methylated strand to orient the recruited methyltransferase correctly to new DNA [12, 13]. 

2. **UHRF1 ablation induces embryonic lethality** with similar features to the effects of DNMT1 loss, including **global hypomethylation** [10, 11]. 

3. **UHRF1 binding to H3K9me2/me3 during S phase is essential for recruitment of this maintenance complex**(see DNMT1 Item 3) [19]. These interactions ensure that DNMT1 activity is stabilized only when in complex with other heterochromatin‐associated proteins and only during DNA replication, providing the fidelity and processivity necessary to recreate a precise, global methylation landscape. 

#### 3) PCNA

1. Proliferating cell nuclear antigen (**PCNA**) is a **DNA** clamp that acts as a processivity factor for **DNA polymerase** δ in eukaryotic cells and is essential for **replication**.  It achieves processivity by encircling the DNA, where it **acts as a scaffold to recruit proteins** involved in DNA replication, DNA repair, chromatin remodeling and epigenetics [14].

### 3.3 DNA methylation dynamics

#### 1) Demethylation in development

1. Both TET1 and TET2, but not TET3, are highly expressed within embryonic stem cells (ESCs) [23], and the genome‐wide binding pattern of TET1 is known [24, 25]. 

2. CpG island proofreading may therefore participate as a general or redundant support during normal development, limiting phenotypes to specific, although essential, developmental transitions (from [1]). 



#### 2) CpG island hypermethylation in aging and  tumorigenesis

1. **CpG islands** are frequently **hypermethylated** during **tumorigenesis**, **ageing** or /in vitro/ culture through a mechanism that probably involves low‐frequency **seeding** of DNA methylation that triggers rapid, terminal heterochromatin formation [20]. Sequences that are hemimethylated /in vitro/often become hypermethylated when integrated into the genome, even when the DNMT3 enzymes are deleted, suggesting that recognition of these seeds by DNMT1 is sufficient to drive the assembly of **heterochromatin**.  Given the dangers of such changes, a number of localized components for catalytic **demethylation** appear to function as a part of CpG‐island‐recognizing epigenetic complexes.

2. Large­scale screen­ ing of cancer or age­related disease cohorts revealed a general trend for regions that are normally regulated by *H3K27* methylation to be frequently *hypermethyl­ated*, suggesting that transcriptionally repressed CpG­ island­ containing genes may be less stably protected from DNA methylation [42]. 

3. Proofreading enzymes, such as the TET dioxygenases, cytosine deaminases and base excision repair (BER) enzymes, are also tightly coupled to regu­ latory complexes that are associated with promoters or enhancers and may prevent infrequent aberrations that could otherwise trigger hypermethylation. (See Figure 1)






### 3.4 DNA methylation enzymes

#### 1) Demethylation enzymes

The TET family of dioxygenases catalyses the oxidation of methylcytosine to hydroxymethylcytosine, which is a potentially crucial intermediate for full reversal to an unmethylated cytosine [22]. 

##### TET1

1. TET1 mostly binds CpG island promoters, including at housekeeping and developmental genes, and probably functions at many enhancers as well [24, 26]. 

2. TET1 recruitment is not altered when DNA methylation, and hence hmC, is completely erased, suggesting that TET1 may bind in larger complexes to function as a general epigenetic proofreader[24, 25]. 


##### TDG

1. Thymine DNA glycosylase (TDG), a base excision repair (BER) protein, recognizes uracil–guanidine and hydroxyuracil–guanidine mismatches that occur at previously methylated cytosines targeted by TET‐mediated oxidation and activation induced cytidine deaminase (AICDA; also known as AID)‐mediated deamination [27].

2. **TDG** and other BER components appear to operate in **complexes** with **MLL** and the histone acetyltransferase **p300**, possibly stabilizing recruitment of these larger complexes to target **promoters** [27, 28]. 

3. Tdg **knockout** is embryonic‐**lethal** , and in these knockouts low‐level methylation **accrues** at **CGI promoters** [27, 28], by contrast, **Aicda**- or **Tet1**‐ **knockout** mice develop to term and appear **overtly normal**. 

## Reference

[1]. Smith, Z. D., & Meissner, A. (2013). DNA methylation: roles in mammalian development. /Nature Reviews Genetics/, /14/(3), 204-220.

[2]. Feng, S., Jacobsen, S. E., & Reik, W. (2010). Epigenetic reprogramming in plant and animal development. /Science/, /330/(6004), 622-627.

[3]. Ziller, M. J., Müller, F., Liao, J., Zhang, Y., Gu, H., Bock, C., … & Gnirke, A. (2011). Genomic distribution and inter-sample variation of non-CpG methylation across human cell types. /PLoS Genet/, /7/(12), e1002389.

[4]. Ramsahoye, B. H., Biniszkiewicz, D., Lyko, F., Clark, V., Bird, A. P., & Jaenisch, R. (2000). Non-CpG methylation is prevalent in embryonic stem cells and may be mediated by DNA methyltransferase 3a. /Proceedings of the National Academy of Sciences/, /97/(10), 5237-5242.

[5]. Okano, M., Bell, D. W., Haber, D. A., & Li, E. (1999). DNA methyltransferases Dnmt3a and Dnmt3b are essential for de novo methylation and mammalian development. /Cell/, /99/(3), 247-257.

[6]. Li, E., Bestor, T. H., & Jaenisch, R. (1992). Targeted mutation of the DNA methyltransferase gene results in embryonic lethality. /Cell/, /69/(6), 915-926.

[7]. Deaton, A. M., & Bird, A. (2011). CpG islands and the regulation of transcription. /Genes & development/, /25/(10), 1010-1022.

[8]. Kishikawa, S., Murata, T., Ugai, H., Yamazaki, T., & Yokoyama, K. K. (2003, September). Control elements of Dnmt1 gene are regulated in cell-cycle dependent manner. In /Nucleic acids symposium series/ (Vol. 3, No. 1, pp. 307-308). Oxford University Press.

[9]. Chuang, L. S. H., Ian, H. I., Koh, T. W., Ng, H. H., Xu, G., & Li, B. F. (1997). Human DNA-(cytosine-5) methyltransferase-PCNA complex as a target for p21WAF1. /Science/, /277/(5334), 1996-2000.

[10]. Sharif, J., Muto, M., Takebayashi, S. I., Suetake, I., Iwamatsu, A., Endo, T. A., … & Tajima, S. (2007). The SRA protein Np95 mediates epigenetic inheritance by recruiting Dnmt1 to methylated DNA. /Nature/, /450/(7171), 908-912.

[11]. Bostick, M., Kim, J. K., Estève, P. O., Clark, A., Pradhan, S., & Jacobsen, S. E. (2007). UHRF1 plays a role in maintaining DNA methylation in mammalian cells. /Science/, /317/(5845), 1760-1764.

[12]. Arita, K., Ariyoshi, M., Tochio, H., Nakamura, Y., & Shirakawa, M. (2008). Recognition of hemi-methylated DNA by the SRA protein UHRF1 by a base-flipping mechanism. /Nature/, /455/(7214), 818-821.

[13]. Avvakumov, G. V., Walker, J. R., Xue, S., Li, Y., Duan, S., Bronner, C., … & Dhe-Paganon, S. (2008). Structural basis for recognition of hemi-methylated DNA by the SRA domain of human UHRF1. /Nature/, /455/(7214), 822-825.

[14]. Moldovan, G. L., Pfander, B., & Jentsch, S. (2007). PCNA, the maestro of the replication fork. /Cell/, /129/(4), 665-679.

[15]. Du, Z., Song, J., Wang, Y., Zhao, Y., Guda, K., Yang, S., … & Sedwick, D. (2010). DNMT1 stability is regulated by proteins coordinating deubiquitination and acetylation-driven ubiquitination. /Science signaling/, /3/(146), ra80-ra80.

[16]. Estève, P. O., Chin, H. G., Benner, J., Feehery, G. R., Samaranayake, M., Horwitz, G. A., … & Pradhan, S. (2009). Regulation of DNMT1 stability through SET7-mediated lysine methylation in mammalian cells. /Proceedings of the National Academy of Sciences/, /106/(13), 5076-5081.

[17]. Wang, J., Hevi, S., Kurash, J. K., Lei, H., Gay, F., Bajko, J., … & Gaudet, F. (2009). The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation. /Nature genetics/, /41/(1), 125-129.

[18]. Fuks, F., Burgers, W. A., Brehm, A., Hughes-Davies, L., & Kouzarides, T. (2000). DNA methyltransferase Dnmt1 associates with histone deacetylase activity. /Nature genetics/, /24/(1), 88-91.

[19]. Rothbart, S. B., Krajewski, K., Nady, N., Tempel, W., Xue, S., Badeaux, A. I., … & Arrowsmith, C. H. (2012). Association of UHRF1 with methylated H3K9 directs the maintenance of DNA methylation. /Nature structural & molecular biology/, /19/(11), 1155.

[20]. Meissner, A., Mikkelsen, T. S., Gu, H., Wernig, M., Hanna, J., Sivachenko, A., … & Gnirke, A. (2008). Genome-scale DNA methylation maps of pluripotent and differentiated cells. /Nature/, /454/(7205), 766-770.

[21]. Lorincz, M. C., Schübeler, D., Hutchinson, S. R., Dickerson, D. R., & Groudine, M. (2002). DNA methylation density influences the stability of an epigenetic imprint and Dnmt3a/b-independent de novo methylation. /Molecular and cellular biology/, /22/(21), 7572-7580.

[22]. Tahiliani, M., Koh, K. P., Shen, Y., Pastor, W. A., Bandukwala, H., Brudno, Y., … & Rao, A. (2009). Conversion of 5-methylcytosine to 5-hydroxymethylcytosine in mammalian DNA by MLL partner TET1. /Science/, /324/(5929), 930-935.

[23]. Ito, S., D’Alessio, A. C., Taranova, O. V., Hong, K., Sowers, L. C., & Zhang, Y. (2010). Role of Tet proteins in 5mC to 5hmC conversion, ES-cell self-renewal and inner cell mass specification. /Nature/, /466/(7310), 1129-1133.

[24]. Williams, K., Christensen, J., Pedersen, M. T., Johansen, J. V., Cloos, P. A., Rappsilber, J., & Helin, K. (2011). TET1 and hydroxymethylcytosine in transcription and DNA methylation fidelity. /Nature/, /473/(7347), 343-348.

[25]. Wu, H., D’Alessio, A. C., Ito, S., Xia, K., Wang, Z., Cui, K., … & Zhang, Y. (2011). Dual functions of Tet1 in transcriptional regulation in mouse embryonic stem cells. /Nature/, /473/(7347), 389-393.

[26]. Stadler, M. B., Murr, R., Burger, L., Ivanek, R., Lienert, F., Schöler, A., … & Tiwari, V. K. (2011). DNA-binding factors shape the mouse methylome at distal regulatory regions. /Nature/, /480/(7378), 490-495.

[27]. Cortellino, S., Xu, J., Sannai, M., Moore, R., Caretti, E., Cigliano, A., … & Abramowitz, L. K. (2011). Thymine DNA glycosylase is essential for active DNA demethylation by linked deamination-base excision repair. /Cell/, /146/(1), 67-79.

[28]. Cortázar, D., Kunz, C., Selfridge, J., Lettieri, T., Saito, Y., MacDougall, E., … & Steinacher, R. (2011). Embryonic lethal phenotype reveals a function of TDG in maintaining epigenetic stability. /Nature/, /470/(7334), 419-423.

[29]. Illingworth, R. S., Gruenewald-Schneider, U., Webb, S., Kerr, A. R., James, K. D., Turner, D. J., … & Bird, A. P. (2010). Orphan CpG islands identify numerous conserved promoters in the mammalian genome. /PLoS Genet/, /6/(9), e1001134.

[30]. Macleod, D., Charlton, J., Mullins, J., & Bird, A. P. (1994). Sp1 sites in the mouse aprt gene promoter are required to prevent methylation of the CpG island. /Genes & development/, /8/(19), 2282-2292.

[31]. Brandeis, M., Frank, D., Keshet, I., Siegfried, Z., Mendelsohn, M., Temper, V., … & Cedar, H. (1994). Spl elements protect a CpG island from de novo methylation. /Nature/, /371/(6496), 435-438.

[32]. Lienert, F., Wirbelauer, C., Som, I., Dean, A., Mohn, F., & Schübeler, D. (2011). Identification of genetic elements that autonomously determine DNA methylation states. /Nature genetics/, /43/(11), 1091-1097.

[33]. Thomson, J. P., Skene, P. J., Selfridge, J., Clouaire, T., Guy, J., Webb, S., … & Turner, D. J. (2010). CpG islands influence chromatin structure via the CpG-binding protein Cfp1. /Nature/, /464/(7291), 1082-1086.

[34]. Clouaire, T., Webb, S., Skene, P., Illingworth, R., Kerr, A., Andrews, R., … & Bird, A. (2012). Cfp1 integrates both CpG content and gene activity for accurate H3K4me3 deposition in embryonic stem cells. /Genes & development/, /26/(15), 1714-1728.

[35]. Erfurth, F. E., Popovic, R., Grembecka, J., Cierpicki, T., Theisler, C., Xia, Z. B., … & Zeleznik-Le, N. J. (2008). MLL protects CpG clusters from methylation within the Hoxa9 gene, maintaining transcript expression. /Proceedings of the National Academy of Sciences/, /105/(21), 7517-7522.

[36]. Otani, J., Nankumo, T., Arita, K., Inamoto, S., Ariyoshi, M., & Shirakawa, M. (2009). Structural basis for recognition of H3K4 methylation status by the DNA methyltransferase 3A ATRX–DNMT3–DNMT3L domain. /EMBO reports/, /10/(11), 1235-1241.

[37]. Ooi, S. K., Qiu, C., Bernstein, E., Li, K., Jia, D., Yang, Z., … & Cheng, X. (2007). DNMT3L connects unmethylated lysine 4 of histone H3 to de novo methylation of DNA. /Nature/, /448/(7154), 714-717.

[38]. Conerly, M. L., Teves, S. S., Diolaiti, D., Ulrich, M., Eisenman, R. N., & Henikoff, S. (2010). Changes in H2A. Z occupancy and DNA methylation during B-cell lymphomagenesis. /Genome research/, /20/(10), 1383-1390.

[39]. Yang, X., Noushmehr, H., Han, H., Andreu-Vieyra, C., Liang, G., & Jones, P. A. (2012). Gene reactivation by 5-Aza-2′-deoxycytidine–induced demethylation requires SRCAP–mediated H2A. Z insertion to establish nucleosome depleted regions. /PLoS Genet/, /8/(3), e1002604.

[40]. Brinkman, A. B., Gu, H., Bartels, S. J., Zhang, Y., Matarese, F., Simmer, F., … & Stunnenberg, H. G. (2012). Sequential ChIP-bisulfite sequencing enables direct genome-scale investigation of chromatin and DNA methylation cross-talk. /Genome research/, /22/(6), 1128-1138.

[41]. Bartke, T., Vermeulen, M., Xhemalce, B., Robson, S. C., Mann, M., & Kouzarides, T. (2010). Nucleosome-interacting proteins regulated by DNA and histone methylation. /Cell/, /143/(3), 470-484.

[42]. Jones, P. A. (2012). Functions of DNA methylation: islands, start sites, gene bodies and beyond. /Nature Reviews Genetics/, /13/(7), 484-492.

[43]. Ginno, P. A., Lott, P. L., Christensen, H. C., Korf, I., & Chédin, F. (2012). R-loop formation is a distinctive characteristic of unmethylated human CpG island promoters. /Molecular cell/, /45/(6), 814-825.

[44]. Schoorlemmer, J., Van Puijenbroek, A., van Den Eijnden, M. M. E. D., Jonk, L., Pals, C., & Kruijer, W. (1994). Characterization of a negative retinoic acid response element in the murine Oct4 promoter. /Molecular and cellular biology/, /14/(2), 1122-1136.

[45]. Dennis, K., Fan, T., Geiman, T., Yan, Q., & Muegge, K. (2001). Lsh, a member of the SNF2 family, is required for genome-wide methylation. /Genes & development/, /15/(22), 2940-2944.

[46]. Zhu, H., Geiman, T. M., Xi, S., Jiang, Q., Schmidtmann, A., Chen, T., … & Muegge, K. (2006). Lsh is involved in de novo methylation of DNA. /The EMBO journal/, /25/(2), 335-345.

[47]. Myant, K., & Stancheva, I. (2008). LSH cooperates with DNA methyltransferases to repress transcription. /Molecular and cellular biology/, /28/(1), 215-226.

[48]. Myant, K., Termanis, A., Sundaram, A. Y., Boe, T., Li, C., Merusi, C., … & Stancheva, I. (2011). LSH and G9a/GLP complex are required for developmentally programmed DNA methylation. /Genome research/, /21/(1), 83-94.

[49]. Epsztejn-Litman, S., Feldman, N., Abu-Remaileh, M., Shufaro, Y., Gerson, A., Ueda, J., … & Bergman, Y. (2008). De novo DNA methylation promoted by G9a prevents reprogramming of embryonically silenced genes. /Nature structural & molecular biology/, /15/(11), 1176-1183.

[50]. Dong, K. B., Maksakova, I. A., Mohn, F., Leung, D., Appanah, R., Lee, S., … & Tachibana, M. (2008). DNA methylation in ES cells requires the lysine methyltransferase G9a but not its catalytic activity. /The EMBO journal/, /27/(20), 2691-2701.

[51]. Ayyanathan, K., Lechner, M. S., Bell, P., Maul, G. G., Schultz, D. C., Yamada, Y., … & Rauscher, F. J. (2003). Regulated recruitment of HP1 to a euchromatic gene induces mitotically heritable, epigenetic gene silencing: a mammalian cell culture model of gene variegation. /Genes & development/, /17/(15), 1855-1869.

[52]. Lehnertz, B., Ueda, Y., Derijck, A. A., Braunschweig, U., Perez-Burgos, L., Kubicek, S., … & Peters, A. H. (2003). Suv39h-mediated histone H3 lysine 9 methylation directs DNA methylation to major satellite repeats at pericentric heterochromatin. /Current Biology/, /13/(14), 1192-1200.

[53]. Chen, T., Tsujimoto, N., & Li, E. (2004). The PWWP domain of Dnmt3a and Dnmt3b is required for directing DNA methylation to the major satellite repeats at pericentric heterochromatin. /Molecular and cellular biology/, /24/(20), 9048-9058.

[54]. Gopalakrishnan, S., Sullivan, B. A., Trazzi, S., Della Valle, G., & Robertson, K. D. (2009). DNMT3B interacts with constitutive centromere protein CENP-C to modulate DNA methylation and the histone code at centromeric regions. /Human molecular genetics/, /18/(17), 3178-3193.

[55]. Okada, T., Ohzeki, J. I., Nakano, M., Yoda, K., Brinkley, W. R., Larionov, V., & Masumoto, H. (2007). CENP-B controls centromere formation depending on the chromatin context. /Cell/, /131/(7), 1287-1300.

[56]. Waterston, R. H., Lindblad-Toh, K., Birney, E., Rogers, J., Abril, J. F., Agarwal, P., … & Antonarakis, S. E. (2002). Initial sequencing and comparative analysis of the mouse genome. /Nature/, /420/(6915), 520-562.

[57].  Liang, G., Chan, M. F., Tomigahara, Y., Tsai, Y. C., Gonzales, F. A., Li, E., … & Jones, P. A. (2002). Cooperativity between DNA methyltransferases in the maintenance methylation of repetitive elements. /Molecular and cellular biology/, /22/(2), 480-491.

[58]. Karimi, M. M., Goyal, P., Maksakova, I. A., Bilenky, M., Leung, D., Tang, J. X., … & Lorincz, M. C. (2011). DNA methylation and SETDB1/H3K9me3 regulate predominantly distinct sets of genes, retroelements, and chimeric transcripts in mESCs. /Cell stem cell/, /8/(6), 676-687.

[59]. Leung, D. C., Dong, K. B., Maksakova, I. A., Goyal, P., Appanah, R., Lee, S., … & Rossi, F. (2011). Lysine methyltransferase G9a is required for de novo DNA methylation and the establishment, but not the maintenance, of proviral silencing. /Proceedings of the National Academy of Sciences/, /108/(14), 5718-5723.

[60]. Ooi, S. K., Wolf, D., Hartung, O., Agarwal, S., Daley, G. Q., Goff, S. P., & Bestor, T. H. (2010). Dynamic instability of genomic methylation patterns in pluripotent stem cells. /Epigenetics & chromatin/, /3/(1), 17.

[61]. Muotri, A. R., Marchetto, M. C., Coufal, N. G., Oefner, R., Yeo, G., Nakashima, K., & Gage, F. H. (2010). L1 retrotransposition in neurons is modulated by MeCP2. /Nature/, /468/(7322), 443-446.

[62]. Jones, P. L., Veenstra, G. C. J., Wade, P. A., Vermaak, D., Kass, S. U., Landsberger, N., … & Wolffe, A. P. (1998). Methylated DNA and MeCP2 recruit histone deacetylase to repress transcription. /Nature genetics/, /19/(2), 187-191.

[63]. Nan, X., Ng, H. H., Johnson, C. A., Laherty, C. D., Turner, B. M., Eisenman, R. N., & Bird, A. (1998). Transcriptional repression by the methyl-CpG-binding protein MeCP2 involves a histone deacetylase complex. /Nature/, /393/(6683), 386-389.

[64]. Jähner, D., Stuhlmann, H., Stewart, C. L., Harbers, K., Löhler, J., Simon, I., & Jaenisch, R. (1982). De novo methylation and expression of retroviral genomes during mouse embryogenesis. /Nature/, /298/(5875), 623-628.

[65]. Stewart, C. L., Stuhlmann, H., Jähner, D., & Jaenisch, R. (1982). De novo methylation, expression, and infectivity of retroviral genomes introduced into embryonal carcinoma cells. /Proceedings of the National Academy of Sciences/, /79/(13), 4098-4102.












